privacy/policies.go

```diff
@@ -0,0 +1,27 @@
+package privacy
+
+import (
+	"github.com/mxmCherry/openrtb"
+
+	"github.com/prebid/prebid-server/privacy/ccpa"
+	"github.com/prebid/prebid-server/privacy/gdpr"
+)
+
+// Policies represents the privacy policies and regulations of an OpenRTB bid request.
+type Policies struct {
+	GDPR gdpr.Policy
+	CCPA ccpa.Policy
+}
+
+// Write mutates an OpenRTB bid request with the context of the policies.
+func (p Policies) Write(req *openrtb.BidRequest) error {
+	if err := p.GDPR.Write(req); err != nil {
+		return err
```

Comment:
Is it necessary to interrupt execution even if only one of them returns an error? After all, they fill two different `Ext` fields.

Reply:
I don't think we should continue with the AMP auction if we can't construct a proper bid request. In reality, this should never occur.